src/app/containers/Headings/index.jsx

```diff
@@ -19,7 +19,7 @@ const HeadingsContainer = ({ blocks, type }) => {
     return null;
   }
 
-  return <Heading>{text}</Heading>;
+  return <Heading text={text}>{text}</Heading>;
```

Comment:
We're passing in the text once as a prop and another time as a child node. Looking at this, I think it would be more optimal to only pass in the text once, as a prop. e.g. `<Heading text={text} />` and use the prop in the component.

Reply:
Sorry, please disregard this comment. Having discussed it with @jtart it seems like it'd be best to keep children as well as a prop, due to future requirements for passing in children that are not just text.